packages/file-explorer/src/FileList.tsx

```diff
@@ -66,21 +71,14 @@ export interface FileListProps {
   overscanCount?: number;
 }
 
-export const getPathFromItem = (file: FileStorageItem): string =>
-  isDirectory(file)
-    ? FileUtils.makePath(file.filename)
-    : FileUtils.getPath(file.filename);
-
-export const DEFAULT_ROW_HEIGHT = 26;
-
 // How long you need to hover over a directory before it expands
-export const DRAG_HOVER_TIMEOUT = 500;
+const DRAG_HOVER_TIMEOUT = 500;
 
 const ITEM_LIST_CLASS_NAME = 'item-list-scroll-pane';
 
-export const renderFileListItem = (
+export function RenderFileListItem(
```

Comment:
Should we just have this in it's own file as well? `FileListItem` ?

Reply:
Probably. I'll move it. First pass was just minimal changes to stop the HMR invalidations